### ISSUE TITLE
Failure on fresh installation of Ubuntu 20.04.1

### ISSUE THREAD

**Issue by yu-yang** [REMnux/remnux-cli#63]
I got the following error when running remnux-cli v1.3.3
>> Running: ssh
Update returned exit code not zero
Error: Update returned exit code not zero
    at ChildProcess.<anonymous> (/snapshot/remnux-cli/remnux-cli.js:569:23)
    at ChildProcess.emit (events.js:315:20)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:286:5)

Sometimes problems occur due to network or server issues when
downloading packages, in which case retrying your operation
a bit later might lead to good results.

I have attached the saltstack.log file. Thanks.
[saltstack.log](https://github.com/REMnux/remnux-cli/files/8105311/saltstack.log)

**Comment by digitalsleuth** [comment]
@yu-yang it looks like there's an SSL error with the files coming from didierstevens.com, indicating that the request is using the wrong TLS version number (error is [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1131)). 

Not sure exactly what's causing the problem at this time, but I'm looking into it. Otherwise, all of the remaining tools in the REMnux suite have installed correctly. The 'checksum' errors you might see in the log as well are also a result of the SSL error, and are not an indication that the hash has changed.

Is it possible that you're on a system which is utilizing a lower version of TLS? Would you be able to run the following command and paste the output of it here:
`openssl ciphers -v | awk '{print $2}' | sort | uniq`

Additionally, it could also be that the new cert from didierstevens.com, issued a couple of weeks ago, could be the culprit. I'll take a look at all factors and keep you posted!

**Comment by yu-yang** [comment]
@digitalsleuth This is the result:
remnux@remnux-VirtualBox:~$ openssl ciphers -v | awk '{print $2}' | sort | uniq
SSLv3
TLSv1
TLSv1.2
TLSv1.3

**Comment by digitalsleuth** [comment]
Thanks @yu-yang. Is there any chance that you're running this behind a proxy of sorts?

**Comment by yu-yang** [comment]
@digitalsleuth I ran remnux from the fresh installation of Ubuntu in virtualbox and not aware of any proxy running.

**Comment by digitalsleuth** [comment]
@yu-yang Could you try using the latest version of Focal, 20.04.3, in the meantime?
Additionally, the proxy I referenced would be on your host machine, or within the network you're utilizing during installation. I haven't been able to duplicate this issue except by forcing my local workstation to use only TLSv1.0 for handshakes, so this would tell me that the issue you're seeing may be related to your host, your network configuration, or the OS.

Since 20.04.3 is the latest version of Focal, this might be the next best, and easiest, way to fault-find the issue. I'll keep working on finding the issue, but this might help narrow down the scope.

Cheers

**Comment by digitalsleuth** [comment]
Also, are you using a VPN?

**Comment by yu-yang** [comment]
@digitalsleuth my host machine is work laptop and does have proxy running. Sorry, I was not aware this could affect the installation in guest machine. Let me try again in my own laptop.

**Comment by digitalsleuth** [comment]
Hi @yu-yang, did you end up trying again on your own laptop? If so, how did it go?

**Comment by yu-yang** [comment]
@digitalsleuth sorry, I was busy yesterday. I tested on my personal laptop without vpn and proxy installed and on Ubuntu 20.04.3 but still got the same issue. Please refer to the new log attached below. Thanks.
[saltstack.log](https://github.com/REMnux/remnux-cli/files/8129778/saltstack.log)

**Comment by digitalsleuth** [comment]
No worries @yu-yang , I took a look at the log and it appears to be the same issue with the same site. So far I'm unable to duplicate the error, so it might take a little while to figure out the cause. Rest assured, I'm looking into it.
